cpp/src/datetime/datetime_ops.cu

```diff
@@ -547,6 +580,58 @@ std::unique_ptr<column> ceil_nanosecond(column_view const& column,
     column, rmm::cuda_stream_default, mr);
 }
 
+std::unique_ptr<column> floor_day(column_view const& column, rmm::mr::device_memory_resource* mr)
+{
+  CUDF_FUNC_RANGE();
+  return detail::floor_general<detail::datetime_component::DAY>(
+    column, rmm::cuda_stream_default, mr);
```

Comment:
will these be updated as well with `detail::round_general` ?

Reply:
yes, just fixed ✔️